### DISCUSSION TITLE
How to change the design & font?

### DISCUSSION THREAD

**Discussion by ghost** [eduardofilo/RG350_adam_image#306] [Ideas & Requests]
Thank you for taking the time to give me some advice.
I'm Sorry. this is the wrong place to post this. I deleted this post.

**Comment by eduardofilo** [comment]
Hi,
The akashi.ttf is inside the SimpleMenu's OPK (in resources folder). You can get it from here if you find it useful:
https://github.com/fgl82/simplemenu/tree/main/simplemenu/output/resources

I don't have much idea about the parameterisation of the themes. I guess the best thing to do is to use the trial/error technique based on the INI definition file of one of the basic themes, like this one:
https://github.com/fgl82/BigCody

You can also try asking in the SimpleMenu or its themes repositories.

About the strange effects of the battery indicator, think that the status is not updated immediately. Sometimes it is necessary to switch screens or even close/open SimpleMenu by launching a game and closing it for SM to update. Again you would have to check in the repository of this application.
